homework-5/P3.java

```diff
@@ -0,0 +1,44 @@
+import java.util.*;
+import java.lang.*;
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+
+class Activity{
+
+    int add(int a, int b){
+        int sum = a + b;
+        return sum;
+    }
+
+
+    boolean checkPallindrome(String s) {
+        int len= s.length();
+        String rev = "";
+
+        for (int i = len-1; i >=0; --i) {
+            rev = rev + s.charAt(i);
+        }
+        return s.toLowerCase().equals(rev.toLowerCase()) ;
+    }
+}
+
+public class P3 {
+    @Test
+    static void unitTesting(){
+        Activity a = new Activity();
+        boolean result1 = a.checkPallindrome("abacaba"); // Should come to TRUE
```

Comment:
segregate your test cases into different methods, each method should only handle 1 test case.